plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -330,8 +330,10 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
 
 				// fundamental resources
-				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes", "pods").RuleOrDie(),
+				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+				rbac.NewRule(append(Read, "delete")...).Groups(legacyGroup).Resources("pods").RuleOrDie(),
```

Comment:
this has the potential to mutate `Read`

Reply:
As far as I know, append does not change the header of the source slice, but I created a separate one for read + delete.